src/main/java/org/opensrp/web/rest/ProductCatalogueResource.java

```diff
@@ -95,7 +106,17 @@
 	public ResponseEntity<String> update(@PathVariable("id") Long uniqueId,
 			@RequestPart(required = false) MultipartFile file,
 			@RequestPart ProductCatalogue productCatalogue) {
+
+		final String DOWNLOAD_PHOTO_END_POINT = "/multimedia/media/";
```

Comment:
can this be a static variable in this class

Reply:
Done